proof/refine/ARM/KHeap_R.thy

```diff
@@ -3502,6 +3500,38 @@ lemma ko_at'_cross:
               split: Structures_A.kernel_object.split_asm if_split_asm
                      ARM_A.arch_kernel_obj.split_asm)
 
+lemma update_sched_context_corres:
```

Comment:
I am not sure if we need this lemma, to be honest... I would think that unfolding `update_sched_context` and using `setSchedContext_no_stack_update_corres` directly for `set_object` and `setSchedContext` should do?

Reply:
Yep, will try this once I've sorted out the `sched_context_unbind_ntfn` rule.

Reply:
Update: Miki and I have decided to keep this in but rename it to `update_sc_no_reply_stack_update_ko_at'_corres` and refactored the immediately following lemma `update_sc_no_reply_stack_update_corres` considerably using this lemma.